meta-freescale-mbl/conf/machine/imx8mmevk-mbl.conf

```diff
@@ -118,9 +118,11 @@ MACHINE_FIRMWARE_remove = " linux-firmware-ath10k"
 ##############################################################################
 MBL_WKS_BOOTLOADER1_FILENAME = "imx-boot-imx8mmevk-mbl-sd.bin-flash_evk"
 MBL_WKS_BOOTLOADER1_OFFSET_BANK1_KiB = "33"
-MBL_WKS_BOOTLOADER1_SIZE_KiB = "1024"
+MBL_WKS_BOOTLOADER1_SIZE_KiB = "991"
 
-MBL_WKS_BOOTLOADER2_SKIP = "1"
+MBL_WKS_BOOTLOADER2_FILENAME = "fip.bin"
+MBL_WKS_BOOTLOADER2_SIZE_KiB = "2048"
```

Comment:
I can see that for other platforms the size is set to `1024`.
Checking the size of the u-boot binary for imx8 it is around 655K.
Can we set `1024` here?

Reply:
...Or maybe I should have set it to a higher value elsewhere.

Reply:
So, what is the decision? Increase to `2048` for other platforms, decrease this one to `1024` or leave as is?

Reply:
fip.bin is 1065937, including optee and u-boot. So 1024K is too small for sure.

Reply:
We should also leave room for future changes... How about we say 4096KiB on all MACHINEs?

Reply:
Or even 16MiB?

Reply:
@jonathanhaigh-arm, no problem. Would you like to raise a ticket to address this?

Reply:
I'll use https://jira.arm.com/browse/IOTMBL-696.